model_definitions/STO/model.json

```diff
@@ -6,13 +6,25 @@
     "source_model_id": "03c9a01f-bd46-4e7c-9a60-4284039097a2",
 	"automation": {
 		"training": {
-			"dockerImage": "artifacts.td.teradata.com/tdproduct-docker-snapshot/avmo/vmo-python-base:3.8.2"
+			"dockerImage": "artifacts.td.teradata.com/tdproduct-docker-snapshot/avmo/vmo-python-base:3.8.2",
+			"resources": {
+				"memory": "700m",
```

Comment:
This way we have a full working example on how to set automations in model code.